Apps/Sandcastle/gallery/3D Tiles.html

```diff
@@ -89,7 +89,7 @@
                     }
                 },
 //              "show": false
-                "show" : "${Height} > 0",
```

Comment:
Instead, can you just comment this out since, as of now, we are using this example to show the different things that can be done with declarative styling, and later we'll break this into separate examples?